src/engine/rendering/gui/GuiManager.cpp

```diff
@@ -0,0 +1,59 @@
+#include "GuiManager.hpp"
+
+#include "components/GuiComponent.hpp"
+#include "core/Application.hpp"
+
+namespace lei3d 
+{
+	GuiManager* GuiManager::s_instance = new GuiManager();
```

Comment:
Does this get deleted?

Reply:
This should exist for the whole runtime of the app, I can't think of a scenario where it would need to be deleted.